src/openforms/js/components/admin/form_design/logic/Today.js

```diff
@@ -9,29 +9,35 @@ import {NumberInput} from '../../forms/Inputs';
 import {getTranslatedChoices} from '../../../../utils/i18n';
 
 
+const EMTPTY_RELATIVE_DELTA = [0, 0, 0];
+
+
 const Today = ({name, value, onChange}) => {
     const sign = value ? jsonLogic.get_operator(value) : '+';
-    const years = value ? value[sign][1]['years'] : 0;
+    var rdelta = value && value[sign][1]?.rdelta ? value[sign][1].rdelta : EMTPTY_RELATIVE_DELTA;
+    if (rdelta.length < 3) {
+        rdelta = [...rdelta].concat(EMTPTY_RELATIVE_DELTA).slice(0, 3);
```

Comment:
This sort of assumes implicitly that `EMPTY_RELATIVE_DELTA` consists of zeroes - which is the case but it's weird to use part of an array. Maybe I'd rather use [`Array.fill`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill) instead of this concat + slice setup.

Also - you could do `[...rdelta, ...EMPTY_RELATIVE_DELTA]` for the same concat results, FYI